lab4/src/src/actions/tabs.js

```diff
@@ -0,0 +1,34 @@
+import store from '../index';
+
+function changeTab(e) {
+    let number;
+    if (e.target.id === 'radioTab1'){
```

Comment:
please create `<Tabs>` and `<Tab>` component to reduce manual switching. Imagine that you can have 30 tabs, would it be possible to create all checks in the same code style?